apps/web/src/components/Nft/NftFeed.tsx

```diff
@@ -1,86 +1,101 @@
 import SingleNft from '@components/Nft/SingleNft';
-import NftsShimmer from '@components/Shared/Shimmer/NftsShimmer';
 import { CollectionIcon } from '@heroicons/react/outline';
-import { t, Trans } from '@lingui/macro';
-import { IS_MAINNET } from 'data/constants';
-import type { Nft, NfTsRequest, Profile } from 'lens';
-import { useNftFeedQuery } from 'lens';
+import { Trans } from '@lingui/macro';
+import { useQuery } from '@tanstack/react-query';
+import axios from 'axios';
+import type { Profile } from 'lens';
 import formatHandle from 'lib/formatHandle';
 import type { FC } from 'react';
-import { useState } from 'react';
-import { useInView } from 'react-cool-inview';
-import { CHAIN_ID } from 'src/constants';
-import { EmptyState, ErrorMessage } from 'ui';
-import { mainnet } from 'wagmi/chains';
+import { useEffect, useState } from 'react';
+import { EmptyState } from 'ui';
+
+import type { NftLinea, RawNfts } from '../../types';
 
 interface NftFeedProps {
   profile: Profile;
 }
 
 const NftFeed: FC<NftFeedProps> = ({ profile }) => {
-  const [hasMore, setHasMore] = useState(true);
+  const [nfts, setNfts] = useState<NftLinea[]>([]);
+
+  const getNfts = async () => {
+    const response = await axios<RawNfts>({
+      url: 'https://api-testnet.nftnest.io/v1/wallet/get_nfts',
+      method: 'POST',
+      data: {
+        address: profile?.ownedBy,
+        chain: 'linea_goerli',
+        page: 0,
+        sort_by: 'minted_newest',
+        contract_addresses: '',
+        name: ''
+      }
+    });
 
-  // Variables
-  const request: NfTsRequest = {
-    chainIds: IS_MAINNET ? [CHAIN_ID, mainnet.id] : [CHAIN_ID],
-    ownerAddress: profile?.ownedBy,
-    limit: 10
+    return response.data;
   };
 
-  const { data, loading, error, fetchMore } = useNftFeedQuery({
-    variables: { request },
-    skip: !profile?.ownedBy
-  });
+  const { data: rawNfts, isLoading: isGalleryLoading } = useQuery([], () => getNfts().then((res) => res));
 
-  const nfts = data?.nfts?.items;
-  const pageInfo = data?.nfts?.pageInfo;
+  useEffect(() => {
+    if (rawNfts) {
+      const collections = Object.entries(rawNfts.contracts['59140']).map((entry) => {
```

Comment:
[optional] Same  as line 49 => could be probably move in a dedicated const using `useMemo`.